src/commonMain/kotlin/com/carterharrison/ecdsa/hash/EcSha256.kt

```diff
@@ -0,0 +1,9 @@
+package com.carterharrison.ecdsa.hash
+
+import org.kotlincrypto.hash.sha2.SHA256
+
+object EcSha256 : EcHasher {
+  override fun hash(data: ByteArray): ByteArray {
+    return SHA256().digest(data)
```

Comment:
Why kotlincrypto?

Reply:
Same. Kotlin stdlib doesn't include any hashing algorithm. We need to include one or reimplement within the library